docs/conf/prague/2019/news/tickets-cfp-reminder.rst

```diff
@@ -0,0 +1,60 @@
+:template: {{year}}/generic.html
+
+.. post:: Mar 26, 2019
+   :tags: prague-2019, tickets, cfp
+
+Announcing Ticket Sales and Call for Proposals Reminder
+=======================================================
+
+Greetings, fellow documentarians in Europe and beyond! We're happy to launch ticket sales for Write the Docs Prague 2019 and give you a quick reminder that our Call for Proposals is still open.
+
+Tickets are on sale!
+--------------------
+
+Ticket prices and descriptions are available on our `Tickets <https://www.writethedocs.org/conf/prague/2019/tickets/>`_ page. Last year we sold out at 280 attendees, and due to venue capacity we are only able to increase our available tickets to 300, to help our attendees feel comfortable in this beautiful venue.
+
+We believe that everyone who cares about documentation and communication has a place at our events, and we keep ticket prices low so that it is accessible to the widest range of people possible.
```

Comment:
```suggestion
We believe that everyone who cares about documentation and communication has a place at our events, and we keep ticket prices low so that the event is accessible to the widest range of people possible.
```